api/zulip/zulip_bot_server.py

```diff
@@ -0,0 +1,99 @@
+from __future__ import absolute_import
+from __future__ import print_function
+from flask import Flask, request, jsonify
+import sys
+import json
+from typing import Any, Dict, Mapping, Union, List
+from werkzeug.exceptions import BadRequest
+from six.moves.configparser import SafeConfigParser
+import optparse
+from importlib import import_module
+
+from zulip import Client
+from bots_api.bot_lib import BotHandlerApi, StateHandler
+
+bots_config = {}  # type: Dict[str, Mapping[str, str]]
+available_bots = []  # type: List[str]
+bots_lib_module = {}  # type: Dict[str, Any]
+
+def read_config_file(config_file_path):
+    # type: (str) -> None
+    parser = SafeConfigParser()
+    parser.read(config_file_path)
+
+    for section in parser.sections():
+        bots_config[section] = {
+            "email": parser.get(section, 'email'),
+            "key": parser.get(section, 'key'),
+            "site": parser.get(section, 'site'),
+        }
+
+def load_lib_modules():
+    # type: () -> None
+    for bot in available_bots:
+        try:
+            package = 'bots.{}'.format(bot)
+            bots_lib_module[bot] = import_module('.' + bot, package=package)
+        except ImportError:
```

Comment:
I think it might be worth it to try this now:
``` python
module_name = 'bots.{bot}.{bot}'.format(bot=bot)
bots_lib_module[bot] = import_module(module_name)
```
I don't know why but an absolute import seems a bit less error-prone to me. :)